usaspending_api/etl/management/commands/reveal_last_period_award_updates.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+from django.db import connection
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from usaspending_api.broker.helpers.last_load_date import get_last_load_date, update_last_load_date
+from usaspending_api.etl.management.helpers.recent_periods import retrieve_recent_periods
+
+logger = logging.getLogger("script")
+
+REVEAL_AWARD_SQL = """
+UPDATE awards
+SET update_date = now()
+WHERE id IN (
+    SELECT DISTINCT faba.award_id
+    FROM
+        financial_accounts_by_awards faba
+    INNER JOIN disaster_emergency_fund_code defc ON
+        defc.group_name = 'covid_19'
+        AND faba.disaster_emergency_fund_code = defc.code
+    INNER JOIN submission_attributes sa ON
+        faba.submission_id = sa.submission_id
+    INNER JOIN dabs_submission_window_schedule dabs ON
+        dabs.id = sa.submission_window_id
+        AND dabs.submission_fiscal_year = {year}
+        AND dabs.submission_fiscal_month = {month}
+        AND dabs.is_quarter = {is_quarter}
+    INNER JOIN awards a on faba.award_id = a.id
+);
+"""
+
+
+class Command(BaseCommand):
+
+    """
+    NOTE: This command is necessary to update award values in Elasticsearch that were modified in the
+    latest submission window when that window's submissions are revealed. Elasticsearch calculations for
+    an award's Covid values do not include data from a submission until its submission window's reveal
+    date has been passed. Because incremental loads in Elasticsearch compare the award's update_date field
+    to a rolling date when determining whether it should be recalculated, the data will not automatically
+    be refreshed after a submission window's reveal date. This command updates the update_date field on
+    awards modified by the latest revealed submission so the Elasticsearch incremental load will include
+    those awards. The command uses the external_data_load_date table to make sure each submission window
+    is only 'revealed' once.
+    """
+
+    def handle(self, *args, **options):
+        script_start_time = timezone.now()
+        periods = retrieve_recent_periods()
+
+        self.last_load_date = get_last_load_date("reveal_last_period_award_updates", default=script_start_time)
```

Comment:
I chose the current time so awards in the latest closed period will not be retouched the first time it is run (I'm making the assumption that this will not be deployed on the specific day that this code needs to be run). The value will be set the first time it is run and work as expected from then on.

Reply:
OK, I understand that. Can you include a brief code comment to mention that and also a log statement to print the datetime it obtains?

Reply:
Yes, added comment and logging